app/src/main/res/values/strings.xml

```diff
@@ -433,4 +433,13 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="additional_contexts_and_projects">Additional contexts and projects</string>
     <string name="enter_space_seperated_contexts_and_projects_these_will">Enter space-separated @contexts and +projects. These will always be visible at todo.txt insert dialogs.</string>
     <string name="infinite">Infinite</string>
+    <string name="search_project">Search projects</string>
+    <string name="search_context">Search contexts</string>
+    <string name="search_category">Search categories</string>
+    <string name="search_tag">Search tags</string>
+    <string name="insert_project">Insert projects</string>
+    <string name="insert_context">Insert contexts</string>
+    <string name="insert_category">Insert categories</string>
+    <string name="insert_tag">Insert tags</string>
+    <string name="clear_selected">Clear selected</string>
```

Comment:
Much better wording. No ambiguity now

Reply:
there is also alternative todo wording. Better "Insert %s"

Reply:
Clear selection